Scripts/GameManagement/SoundManager.gd

```diff
@@ -0,0 +1,54 @@
+extends Node
+
+var win_sound
+
+func _ready():
+	win_sound = AudioStreamPlayer.new()
+	add_child(win_sound)
+	win_sound.stream = load("res://Sounds/spinigeddonWinSound.wav")
+	win_sound.volume_db = -20
+
+remote func play_moving_sound(player_name):
+	if GameManager.in_game == true:
+		if !GameManager.world.get_node_or_null(player_name + "/Sounds/MovingSound").playing and get_tree().get_network_unique_id() != int(player_name):
+			GameManager.world.get_node(player_name + "/Sounds/MovingSound").play()
+		
+remote func stop_moving_sound(player_name):
+	GameManager.world.get_node(player_name + "/Sounds/MovingSound").stop()
+
+remote func play_shoot_sound(player_name):
+	GameManager.world.get_node(player_name + "/Sounds/ShootSound").play()
+
+remote func play_hit_material_sound(object_name): #doesnt work i dont know why (maybe because bullet is deleted) but i think it is not neccessary
+	GameManager.world.map.get_node(object_name + "/Sounds/HitMaterialSound").play()
```

Comment:
Do we need this in the code then?

Reply:
Yeah this comment is obsolete, now it works, because material sends signal to play sound if it detects collision, no bullet.